domains/aadarsh.json

```
@@ -0,0 +1,8 @@
+{
+  "owner": {
+    "email": "aadarsh.rai0307@gmail.com"
+  },
+  "record": {
```

```suggestion
  "records": {
```

it must be `records` not `record`